ci-operator/config/openshift/assisted-installer-agent/openshift-assisted-installer-agent-ocm-2.3.yaml

```diff
@@ -71,8 +71,6 @@ tests:
       ASSISTED_AGENT_IMAGE: pipeline:assisted-installer-agent
     env:
       ASSISTED_CONFIG: |
-        SERVICE_BRANCH=ocm-2.3
-        INSTALLER_BRANCH=ocm-2.3
```

Comment:
So removing these causes assisted-test-infra to pull the master branch for these repos. There is now an extremely a large difference between the code bases at this point I don't recommend running assisted-test-infra against the master branches of these repos.

Reply:
I agree with you on that. Will fix.

I would expect `PULL_BASE_REF` to be `ocm-2.3`
But since it's a rehease job, this value isn't overriden at all and the default is used


https://github.com/openshift/release/blob/e10d36d040ab63826250aa6451f297f1d9eba99c/ci-operator/step-registry/baremetalds/assisted/setup/baremetalds-assisted-setup-commands.sh#L79-L87



```
    - name: PULL_BASE_REF
      value: master
```

Reply:
Ah that is what is happening with the rehearse job. Makes sense. Yeah I would expect `PULL_BASE_REF` need to be updated to `ocm-2.3` to get the correct base for these tests.